.github/main.workflow

```diff
@@ -1,17 +1,27 @@
 workflow "New workflow" {
   on = "push"
-  resolves = ["./scripts/publish-manager.sh"]
+  resolves = ["push"]
 }
 
 action "Docker Registry" {
   uses = "actions/docker/login@86ff551d26008267bb89ac11198ba7f1d807b699"
+  secrets = ["DOCKER_USERNAME", "DOCKER_PASSWORD", "DOCKER_REGISTRY_URL"]
```

Comment:
Are these injected automatically?

Reply:
oh no, they are not. I had to create the secrets as an admin. There is a better way to do it with gpg which we should probably look into and use a robot user for these values instead of my personal account

Reply:
https://developer.github.com/actions/managing-workflows/storing-secrets/

Reply:
oh, that's disappointing, it would be nice if there was cleaner integration with github package repository.

Reply:
Can the [GitHub Token secret](https://developer.github.com/actions/managing-workflows/storing-secrets/#github-token-secret) be used for this purpose instead of adding additional secrets?